packages/storybook/package.json

```diff
@@ -7,8 +7,8 @@
     "build": "yarn copy-changelog && build-storybook",
     "clean": "rm -rf storybook-static src/stories/changelog",
     "copy-changelog": "mkdir -p src/stories/changelog && ts-node scripts/copy-changelog",
-    "lint": "eslint .",
-    "lint:fix": "yarn lint --fix",
+    "lint": "eslint . && tsc --noEmit",
```

Comment:
I realized Storybook wasn't throwing TypeScript compilation errors during build time so I added it here